addons/magazinerepack/stringtable.xml

```diff
@@ -29,7 +30,7 @@
         </Key>
         <Key ID="STR_ACE_MagazineRepack_timePerBeltLink">
             <English>Time per belt link</English>
-            <German>Zeit pro Gurtglied</German>
+            <German>Zeit je Gurtglied</German>
```

Comment:
warum "je" statt "pro"

Reply:
"Zeit für jedes Gurtglied" would be the complete name.
Both variants don't sound perfect to me.